### PR TITLE
Add example of loading a font from .ttf file

### PR DESCRIPTION
**Pull request by follower**
As this is a multi-step process it would be nice to have this documented.

### CODE REVIEW

File: doc/classes/DynamicFont.xml [hunk 1]
```diff
@@ -5,6 +5,12 @@
 	</brief_description>
 	<description>
 		DynamicFont renders vector font files (such as TTF or OTF) dynamically at runtime instead of using a prerendered texture atlas like [BitmapFont]. This trades the faster loading time of [BitmapFont]s for the ability to change font parameters like size and spacing during runtime. [DynamicFontData] is used for referencing the font file paths.
+		[codeblock]
+		var dynamic_font = DynamicFont.new()
+		dynamic_font.font_data = load("res://BarlowCondensed-Bold.ttf")
```
Comment (volzhs) [inline doc/classes/DynamicFont.xml]: `load("res://path/to/font.ttf")` would be better for general.

Comment (Calinou) [reply doc/classes/DynamicFont.xml]: Shouldn't we also use `preload()` instead of `load()` in the example?

Comment (volzhs) [reply doc/classes/DynamicFont.xml]: I think this example can be used as loading font on demand.
so, `load()` seems fine to me.

**Comment by hpvb** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.2.

**Pull request merged**